web/src/title/loader.ts

```diff
@@ -45,51 +49,63 @@ export async function loadFromLocalStorage(key: string) {
   }
 }
 
-// Returns OSM input and the boundary polygon, either from a pre-hosted pbf
-// file or from Overpass.
-async function getOsmInput(
+// Returns OSM input, optional demand model input, and the boundary polygon,
+// either from pre-hosted files or from Overpass.
+async function getInputFiles(
   gj: any,
   isCnt: boolean,
-): Promise<[ArrayBuffer, Feature<Polygon>]> {
+): Promise<[ArrayBuffer, ArrayBuffer | null, Feature<Polygon>]> {
   if (isCnt) {
     // CNT projects are stored in a different place
     let url1 = get(useLocalVite)
       ? `/cnt_osm/${gj.study_area_name}.osm.pbf`
       : `https://assets.od2net.org/cnt_osm/${gj.study_area_name}.osm.pbf`;
     console.log(`Grabbing ${url1}`);
     let resp1 = await safeFetch(url1);
-    let bytes = await resp1.arrayBuffer();
+    let osmBytes = await resp1.arrayBuffer();
 
     let url2 = get(useLocalVite)
       ? `/cnt_boundaries/${gj.study_area_name}.geojson`
       : `https://assets.od2net.org/cnt_boundaries/${gj.study_area_name}.geojson`;
     let resp2 = await safeFetch(url2);
     let boundary = await resp2.json();
 
-    return [bytes, boundary];
+    let url3 = get(useLocalVite)
+      ? `/cnt_demand/demand_${gj.study_area_name}.bin`
+      : `https://assets.od2net.org/cnt_demand/demand_${gj.study_area_name}.bin`;
+    console.log(`Grabbing ${url3}`);
+    let demandBytes = null;
+    try {
+      let resp3 = await safeFetch(url3);
+      demandBytes = await resp3.arrayBuffer();
+    } catch (err) {
+      console.log(`No demand model: ${err}`);
+    }
+
+    return [osmBytes, demandBytes, boundary];
   } else if (gj.study_area_name) {
     let url1 = get(useLocalVite)
       ? `/osm/${gj.study_area_name}.pbf`
       : `https://assets.od2net.org/severance_pbfs/${gj.study_area_name}.pbf`;
     console.log(`Grabbing ${url1}`);
     let resp1 = await safeFetch(url1);
-    let bytes = await resp1.arrayBuffer();
+    let osmBytes = await resp1.arrayBuffer();
 
     let url2 = get(useLocalVite)
       ? `/boundaries/${gj.study_area_name}.geojson`
       : `https://assets.od2net.org/boundaries/${gj.study_area_name}.geojson`;
     let resp2 = await safeFetch(url2);
     let boundary = await resp2.json();
 
-    return [bytes, boundary];
+    return [osmBytes, null, boundary];
   } else {
     console.log(`Grabbing from Overpass`);
     let study_area_boundary = gj.features.find(
       (f: Feature) => f.properties!.kind == "study_area_boundary",
     )!;
     let resp = await safeFetch(overpassQueryForPolygon(study_area_boundary));
     let bytes = await resp.arrayBuffer();
```

Comment:
```suggestion
let osmBytes = ...
```

Reply:
Looks like a typo/mistaken suggestion?

Reply:
Sorry - I was being too concise. It was a minor nit anyway, so I just pushed up a commit: bf22d137a7c219f0b2aabac85ab40f8a9b4448e9